docs/strategy_migration.md

```diff
@@ -192,7 +192,7 @@ class AwesomeStrategy(IStrategy):
         return False 
 ```
 
-### Custom-stake-amount
+### `custom_stake_amount`
```

Comment:
I'm not against this change in general - but please also update the link in the index of this page (that should be the only occurance) - it'd be broken with this PR - so navigation to this point would no longer work.